src/main/scala/io/confluent/examples/streams/algebird/CMSStoreChangeLogger.scala

```diff
@@ -44,7 +44,7 @@ class CMSStoreChangeLogger[K, V](val storeName: String,
     if (collector != null) {
       val keySerializer = serialization.keySerializer
       val valueSerializer = serialization.valueSerializer
-      collector.send(this.topic, key, value, this.partition, context.timestamp, keySerializer, valueSerializer)
+      collector.send(this.topic, key, value, this.partition, null, keySerializer, valueSerializer)
```

Comment:
For my understanding: why we need this change?

Reply:
This we change result timestamp semantics to "processing time" as the producer will assign timestamp as current system time. Not sure if we want to have this in this example? We should also pass in the timestamp as parameter.

@miguno We cannot call `context.timestamp` in flush -- only if we process records.

Reply:
Yeah, it is because we can't do `context.timestamp` in flush

Reply:
We don't have the timestamp to pass in as it is called from flush. I guess we could save the timestamp on each put and use that

Reply:
Though i'm not sure it actually matters in this example as it is only ever a single object that is written out

Reply:
It could be the timestamp of the record that did the last update.